provider/ansible/request.rb

```diff
@@ -128,28 +128,30 @@ def request_output(prop, hash_name, module_name)
             "#{hash_name}.get(#{quote_string(prop.out_name)}, [])",
             ", #{module_name}).to_request()"
           ].join
-        elsif prop.is_a?(Api::Type::ResourceRef) && !prop.resource_ref.virtual
+        elsif prop.is_a?(Api::Type::ResourceRefs) && \
```

Comment:
Should this have the same comment as the other file about assuming all of them have the same virtual value?